infra/resources/prod/locals.tf

```diff
@@ -7,9 +7,12 @@ locals {
   project            = "${local.prefix}-${local.env_short}-${local.location_short}"
   weu_project        = "${local.prefix}-${local.env_short}-${local.weu_location_short}"
   domain             = "auth"
+  full_domain        = "citizen-auth"
```

Comment:
```suggestion
  legacy_domain      = "citizen-auth"
```

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/132/commits/e4d5135b95b33dec8ea56597d13eb3f56032dca0